lib/publiccloud/azure.pm

```diff
@@ -43,6 +43,7 @@ sub init {
     $self->SUPER::init();
     $self->vault_create_credentials() unless ($self->key_id);
     $self->az_login();
+    assert_script_run("az account set --subscription c011786b-59d7-4817-880c-7cd8a6ca4b19");
```

Comment:
Shouldn't we keep that as parameter so it's not exposed in the public repo? I miss expertise in azure code we have, but I believe `$self->subscription` will contain value we need.

Reply:
I totally forgot about that :/ Changing it right away.